cypress/components/sidebar/sidebar.cy.js

```diff
@@ -297,4 +301,50 @@ context("Testing Sidebar component", () => {
       cy.checkAccessibility();
     });
   });
+
+  describe("test background scroll when tabbing", () => {
+    it("tabbing forward through the sidebar and back to the start should not make the background scroll to the bottom", () => {
+      CypressMountWithProviders(<SidebarBackgroundScrollTestComponent />);
+
+      continuePressingTABKey(3);
+
+      closeIconButton().should("be.focused");
+
+      cy.checkNotInViewport("#bottom-box");
+    });
+
+    it("tabbing backward through the sidebar and back to the start should not make the background scroll to the bottom", () => {
+      CypressMountWithProviders(<SidebarBackgroundScrollTestComponent />);
+
+      continuePressingTABKey(2, true);
+
+      closeIconButton().should("be.focused");
+
+      cy.checkNotInViewport("#bottom-box");
+    });
+
+    it("tabbing forward through the sidebar and other focusable containers back to the start should not make the background scroll to the bottom", () => {
+      CypressMountWithProviders(
+        <SidebarBackgroundScrollWithOtherFocusableContainers />
+      );
+
+      continuePressingTABKey(6);
+
+      closeIconButton().eq(0).should("be.focused");
+
+      cy.checkNotInViewport("#bottom-box");
+    });
+
+    it("tabbing backward through the dialog and other focusable containers back to the start should not make the background scroll to the bottom", () => {
```

Comment:
it("tabbing backward through the sidebar and other focusable containers back to the start should not make the background scroll to the bottom", () => {